### DISCUSSION TITLE
Healing Properties of Human Saliva人类唾液的治疗特性【Siyu Li】

### DISCUSSION THREAD

**Discussion by linsuanyanleisiyulaoshuyao** [realitydeslab/playshop-summer-2024#17] [Proposal]
paper
[Group-PDF-Export.pdf](https://github.com/user-attachments/files/16093573/Group-PDF-Export.pdf)
3d建模
[Healing Properties of Human Saliva(3).zip](https://github.com/user-attachments/files/16093586/Healing.Properties.of.Human.Saliva.3.zip)

25词的口号：口水！口水！我们需要口水！请向我吐口水！我们将他制成药！他将在未来造福于人类！
 Saliva! Saliva! We need saliva! Please spit on me! We made him into medicine! He will benefit mankind in the future!
视频描述：在城市的大街上，一个男人捂嘴微微一笑，走向前方的口水收集装置吐了一口痰，笑着走开了
On the street of the city, a man covered his mouth and smiled. He went to the saliva collection device in front of him and spit out phlegm and walked away with a smile.
研究问题：在未来唾液具有医疗作用，会发生什么
Research Questions: What will happen if saliva has a medical effect in the future?
关键词：唾液，医疗，社会行为，文明行为，燕窝，痰盂
Keywords: Saliva, medical treatment, social behavior, civilized behavior, bird's nest, [spittoon](url)
实现手法：人工制品，模型，3D打印，装置
Artifact , media : Model
摘要：探究人类吐口水历史，唾液是一种无色且稀薄的液体，被人们俗称为口水，虽然在古代被称为“金津玉液”，现代却向来给人有不洁不雅之感。发现人类在古代就有此行为，皇宫贵族将唾液吐在名为“痰盂”或者“唾壶”的容器中，随着社会进步和生活的改善，痰盂逐渐被现代化设施所取代，成为过时的老物件，而现代也有吐在垃圾桶或者地上的行为。如果人类需要唾液，重新利用痰盂，那么在未来痰盂会是什么形象。现有研究唾液酸，唾液酸结合免疫球蛋白样凝集素，肿瘤b免疫逃逸，髓系免疫细胞。唾液中含有多种治疗作用的物质，可以止血，抗菌，治伤，消化，抗衰。像现有的燕窝就是用燕子的唾液混合其他物质所筑成的巢穴制成的。唾液作为每个人都有，本没有价值的东西，如果在未来，赋予唾液价值，唾液具有医疗作用，交换唾液也会让两个人越来越相似，甚至可以改变外貌，延长寿命。一些人类的“必需品”，必须冠以包装才能让人重视。成为药物之后，向别人吐口水还会是侮辱人的行为吗。唾液变的有价值之后，还会出现随地吐口水的行为吗。随地吐口水在当今社会是不文明行为，那么在未来唾液具有价值，吐口水行为会不会变成文明行为。
abstract：Exploring the history of human spitting, Saliva is a colorless and thin liquid, commonly known as saliva. Although it was called "Jinjin jade liquid" in ancient times, it has always given people a sense of indecentness in modern times. We found that human beings had this behavior in ancient times. The nobles of the palace spit saliva in a container called "sputtoon" or "spit pot". With social progress and the improvement of life, the spittoon was gradually replaced by modern facilities and became outdated old objects, while modern people also spit in garbage cans or on the ground. Behavior. If humans need saliva and reuse the spittoon, what will the spittoon be like in the future? At present, sialic acid, sialic acid combined with immunoglobulin-like lectin, tumor immune escape, and medullary immune cells have been studied. Saliva contains a variety of therapeutic substances, which can stop bleeding, antibacterial, treat injuries, digestion and anti-aging. For example, the existing bird's nest is made of nests made of swallows' saliva mixed with other substances. Saliva is something that everyone has and has no value. If in the future, saliva is given value and saliva has a medical effect, the exchange of saliva will also make two people more and more similar, and even change their appearance and prolong their life. Some human "essentials" must be packaged to be valued. After becoming a drug, will spitting on others be an insult? After the saliva becomes valuable, will there be the behavior of spitting everywhere? Spitting everywhere is uncivilized behavior in today's society, so saliva has value in the future. Will spitting become a civilized behavior?
文献：
https://link.zhihu.com/?target=https%3A//www.zhangqiaokeyan.com/academic-journal-cn_journal-nantong-university-medical-sciences_thesis/0201288799497.html%3Ffrom%3D01-010-04-z-106946
关键帧：
![C8894DD3-4154-42D4-AD61-DFD438A26F87](https://github.com/realitydeslab/playshop-summer-2024/assets/173657653/91f348ad-9e53-4c7a-811f-09052b2b952c)
![C81A5105-B61E-4C88-8955-55675B423AD7](https://github.com/realitydeslab/playshop-summer-2024/assets/173657653/7bd4e009-7c4b-434e-95d9-b09c5becc7cd)
![AABB6E3E-6D15-4033-858D-92800526150C](https://github.com/realitydeslab/playshop-summer-2024/assets/173657653/8fca9527-1abf-456f-88dd-84e621fd0737)
![0EAF941A-509A-4D9C-AD05-D13029958285](https://github.com/realitydeslab/playshop-summer-2024/assets/173657653/b9295f4d-26f4-485f-959b-913343ba21dc)

**Comment by botaohu** [comment]
bird saliva collecting

**Comment by glitch-kraft** [comment]
**Title**
Healing Properties of Human Saliva. Chinese Spitting Culture. 

**Tagline**
Research into the healing properties of human saliva.To restore Chinese spitting in public culture through modernised spittoons designed to collect valuable biological material for further applications for the public good.
